tests/playgrounds/advanced_playground.html

```diff
@@ -22,6 +22,7 @@
       function start() {
         setBackgroundColour();
         initPlayground();
+        workspace.addChangeListener(Blockly.Events.disableOrphans);
```

Comment:
We only needed this change in order to test the code you wrote above. We don't want to commit this change to the playground, so you should revert the changes to this file.

Reply:
sure, I have made those changes